### DISCUSSION TITLE
Have to restart publishers

### DISCUSSION THREAD

**Discussion by madgrizzle** [eProsima/Fast-DDS#3882] [Support]
I'm using FastDDS on Rolling and I run about 30 published topics on my robot and I leave the robot on all the time.  I find that often when I restart one of the nodes after its been running for a while, it doesn't resume receiving messages on the topic it subscribes to.  The only solution I've found is to also restart the publisher and then the subscriber starts receiving them.  If I don't restart it, it continues to run fine receiving messages.  I'm at a bit of a loss as to what to do or look for.

**Comment by JesusPoderoso** [comment]
Hi @madgrizzle 
This issue was fixed in [Fast DDS relase v2.11.2](https://fast-dds.docs.eprosima.com/en/v2.11.2/notes/notes.html), so make sure your Rolling branch is updated. If that is the case, could you let us know the last commit / version you were using on Rolling?
